src/Microsoft.Identity.Client/Platforms/Mac/AuthenticationAgentNSWindowController.cs

```diff
@@ -247,7 +247,7 @@ public void FinishedLoad(WebView sender, WebFrame forFrame)
 
         void CancelAuthentication()
         {
-            _callbackMethod(new AuthorizationResult(AuthorizationStatus.UserCancel, null));
+            _callbackMethod(new AuthorizationResult(AuthorizationStatus.UserCancel));
```

Comment:
Since you have a factory for "FromUri" above, should you have a "FromStatus" here as well for construction consistency?